packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -23,49 +23,99 @@ import './page.css';
 
 ## Examples
 
-### Vertical nav
+### Basic page components
+
+A page will typically contain the following components: 
+
+- A `<Page>` with a `header` that often contains a [masthead](/components/masthead)
+  - Mastheads contain the `<PageToggleButton>`, a `<MastheadMain>` that contains a `<MastheadBrand>`, and the page's header toolbar within `<MastheadContent>`. 
+- A `<PageSidebar>` for vertical navigation 
+- One or more `<PageSection>` components
+
+### Vertical navigation
+
+To add a vertical sidebar to a `<Page>`, pass a `<PageSidebar>` component into the `sidebar` property. To render navigation in the sidebar, use the `nav` property of the `<PageSidebar>`. 
+
+The `isNavOpen` property helps facilitate the opening and closing of the sidebar and should be 'true' when the navigation sidebar is visible. 
 
 ```ts file="./PageVerticalNav.tsx"
 ```
 
-### Horizontal nav
+### Legacy page header
+
+This example shows the legacy implementation of a page's vertical navigation. Our updated recommendation advises you to use a masthead and toolbar to make headers, rather than `<PageHeader>` and `<PageHeaderTools>` as shown in the following example.
+
+```ts file="./PageVerticalNavUsingPageHeaderComponent.tsx"
+```
+
+### Horizontal navigation
+
+To add horizontal top navigation to a `<Page>`, each navigation item should be set as a `<ToolbarItem>` in the `<Toolbar>` that is passed to the `<MastheadContent>` of the `<Masthead>`.
 
 ```ts file="./PageHorizontalNav.tsx"
 ```
 
-### Tertiary nav
+### Legacy tertiary navigation
+
+Horizontal sub-navigation is now recommended instead of tertiary-level navigation. 
+
+Tertiary navigation allows you to add an additional navigation menu alongside vertical or horizontal navigation. To create tertiary navigation, use the `tertiaryNav` property. The following example passes `tertiaryNav="Navigation"` into a `<Page>` component.
```

Comment:
FWIW it can also be created via

```
<PageNavigation>[nav]</PageNavigation>
<PageSection type="nav">[nav]</PageSection>
```

Reply:
Good to know! Is this a common implementation? Generally, I haven't mentioned alternative implementation options unless they're present within the example. So (unless I'm missing this in another example) I wonder if this is worth adding a whole new example?

Reply:
I think it's fine for now. cc @tlabaj in case she'd like to add that. I can add it in https://github.com/patternfly/patternfly-react/pull/8485 if we want to add it.

Reply:
I do not think it is worth adding another example since tertiary nav is not recommended.